lib/assessment.md

```diff
@@ -163,5 +184,5 @@ Consider the following media queries. How wide would the screen need to be for e
 ```
 
 <!-- your answer starts here -->
-
+1200px - 400px x 3!
```

Comment:
At 1200px width, both media queries are 'visible', and since CSS cascades, the last rule is the one that wins out. 30% of 1200px is 360px, not 150px.